src/layout/Menu.tsx

```diff
@@ -33,15 +34,14 @@ const MENU_ITEMS: ItemType[] = [
 
 const BOTTOM_MENU: ItemType[] = [
   {
-    label: 'Tools',
-    icon: <BiCog className="text-xl" />,
-    url: '',
-    submenu: [
-      {
-        label: 'Commodities',
-        url: '/dashboard/commodities',
-      },
-    ],
```

Comment:
changed this coz couldn't find a quick way to show the menu to top direction and going bottom makes the menu to flicker (coz larger than the space left at the bottom of the page)